The morse module uses DEV_COREDUMP, but because it's out of
tree it can't tell the kernel about it.

diff --git a/drivers/base/Kconfig b/drivers/base/Kconfig
index ffcbe2bc460e..215a5dad5555 100644
--- a/drivers/base/Kconfig
+++ b/drivers/base/Kconfig
@@ -94,6 +94,7 @@ source "drivers/base/firmware_loader/Kconfig"
 
 config WANT_DEV_COREDUMP
 	bool
+	default y
 	help
 	  Drivers should "select" this option if they desire to use the
 	  device coredump mechanism.
